src/recoil/app-theme/app-theme.state.ts

```diff
@@ -0,0 +1,29 @@
+import { atom } from 'recoil'
+import { Rkp } from '@/recoil/index.keys'
+import { DateTime } from 'luxon'
+
+/**
+ * Manage the state of the app theme to mimic the GitHub's Halloween theme
+ */
+
+export enum AppTheme {
+  Basic = `basic`,
+  Halloween = `halloween`,
+}
+
+const halloweenStart = DateTime.fromObject({ month: 10, day: 17 }).startOf(
+  `day`,
+)
+const halloweenEnd = DateTime.fromObject({ month: 11, day: 1 }).endOf(`day`)
```

Comment:
Just define one halloweenDate, and maybe just have the span of two weeks, with one extra day & the rest is 13